---
layout: default
title: Nielsen Lab / projects
categories:
 - projects
---

    <h1>Methods for Inference of Demography and Selection</h1>
    <hr style="height:3px;border:none;color:#808088;background-color:#808088;" />
    <img src="/images/Picture1.png" style="float:right">

    A core line of research in the group is the development of computational methods for inference of complex demographic models.  For example, we have previously
     developed Markov Chain Monte Carlo (MCMC) methods for joint estimation of population divergence times,
      changes in population size, gene-flow between populations,
      and other parameters together with Jody Hey at Temple University and implemented in the
      <a href="https://bio.cst.temple.edu/~tuf29449/software">IM programs</a>.  We have also developed a number of methods
      for detecting natural selection from DNA sequence data, including the methods implemented in
      <a href="http://degiorgiogroup.fau.edu/sf2.html">SweepFinder2</a>,
      <a href="http://degiorgiogroup.fau.edu/vf.html">VolcanoFinder</a>  and
      <a href="https://github.com/jade-cheng/ohana">Ohana</a>.
    We are currently focusing on developing rigorous methods for using Ancestral Recombination Graphs for
     inferences of demography and selection, so far resulting in the methods implemented in
     <a href="https://github.com/35ajstern/clues">CLUES</a> and
     <a href="https://github.com/35ajstern/palm">PALM</a>.

     <div class="bigspacer"></div>

     <h4>Example Papers</h4>
        <ol type = "1">
          <li><a href="/pdfs/papers/disentangle.pdf">Disentangling selection on genetically correlated polygenic traits via whole-genome genealogies</a></li>
          <li><a href="/pdfs/papers/clues.pdf">An approximate full-likelihood method for inferring selection and allele frequency trajectories from DNA sequence data</a></li>
          <li><a href="/pdfs/papers/joint.pdf">Joint Estimation of Pedigrees and Effective Population Size Using Markov Chain Monte Carlo</a></li>
          <li><a href="/pdfs/papers/pulses.pdf">Estimating the timing of multiple admixture pulses during local ancestry inference</a></li>
         </ol>
    <div class="bigspacer"></div>


    <h1>Bioinformatics for Next Generation Sequencing Data</h1>
    <hr style="height:3px;border:none;color:#808088;background-color:#808088;" />
    <img src="/images/Picture2.png" style="float:right">

    Over the past 15 years we have developed a number of statistical and computational
    tools used in analyses and interpretations of NGS data.  Many of the methods are made
    available to other researchers in the <a href="http://www.popgen.dk/angsd/index.php/ANGSD">ANGSD</a> package.
    These methods are designed for non-model organisms for which imputation-based genotype calling is unreliable
    or impossible.  Instead, we model the uncertainty in the genotype calls using genotype likelihoods and
    incorporate genotype uncertainty in downstream analyses.  This approach allows researchers to leverage
     inexpensive low-coverage sequencing data for advanced population genetic analyses.
     <div class="bigspacer"></div>
     <h4>Example Papers</h4>

     <ol type = "1">
      <li><a href="/pdfs/papers/fastadm.pdf">Fast admixture analysis and population tree estimation for SNP and NGS data</a></li>
      <li><a href="/pdfs/papers/lowcov.pdf">Estimating IBD tracts from low coverage NGS data</a></li>
      <li><a href="/pdfs/papers/agsd.pdf">ANGSD: Analysis of Next Generation Sequencing Data</a></li>
      </ol>
 <div class="bigspacer"></div>
 <div class="bigspacer"></div>
 <div class="bigspacer"></div>
 <div class="bigspacer"></div>
 <div class="bigspacer"></div>
 <div class="bigspacer"></div>




    <h1>Human Genetic Diversity and Disease</h1>
    <hr style="height:3px;border:none;color:#808088;background-color:#808088;" />
    <img src="/images/Picture3.png" style="float:right">

    Most human genetic studies have focused on European and other highly represented populations.
     However, much of human genetic and phenotypic variation is not represented in this group.
     Focusing primarily on Europeans limits our ability to provide equal medical treatment for all
     population groups.  Furthermore, understudied populations harbor variants not commonly found in
      Europeans that can help provide a better understanding of disease etiology for the benefit of all.
     My group has specialized in analyzing genetic variation in understudied groups, including Inuit
     from Greenland, Tibetans, Aboriginal Australians, Native Americans, and other groups.  For example,
      we have shown that Tibetans harbor unique genetic variants that modulate the regulation of erythropoiesis
      to avoid mis-regulation in high altitude.  We have also shown that these variants were transferred to
      Tibetans by interbreeding with extinct hominids called ‘Denisovans’.
      In studies of Aboriginal Australians we have identified unique genetic adaptations associated with
      thyroid function, likely related to thermal regulation.  In Inuit from Greenland, we have identified
       mutations that alter the regulation of fatty acid desaturase (FADS) genes, thereby decreasing the endogenous
        synthesis of certain poly-unsaturated fatty acids (PUFAs) to compensate for a high dietary intake of these fatty acids.
        <div class="bigspacer"></div>
        <h4>Example Papers</h4>

        <ol type = "1">
          <li><a href="/pdfs/papers/disvar.pdf">Human Disease Variation in the Light of Population Genomics</a></li>
            <li><a href="/pdfs/papers/extenv.pdf">Human adaption to extreme environmental conditions. Current opinion in genetics & development</a></li>
              <li><a href="/pdfs/papers/nomads.pdf">Physiological and Genetic Adaptations to Diving in Sea Nomads</a></li>
                <li><a href="/pdfs/papers/andeans.pdf">Natural selection on genes related to cardiovascular health in high-altitude adapted Andeans</a></li>
                  <li><a href="/pdfs/papers/tracing.pdf">Tracing the peopling of the world through genomics</a></li>
         </ol>
    <div class="bigspacer"></div>







       <h1>Analyses of Ancient DNA</h1>
       <hr style="height:3px;border:none;color:#808088;background-color:#808088;" />
       <img src="/images/Picture4.png" style="float:right">

       We work together with Eske Wilerslev and other researchers at
       the <a href="https://globe.ku.dk/research/lundbeck-foundation-geogenetics-centre/">Lundbeck Foundation Center for GeoGenetics</a> at the University of Copenhagen
        on analyzing ancient DNA.  The questions are broad but include patterns of origins and dispersal of
         humans and our adaptation to local environments. We have particularly worked on the peopling of the
          Americas and on European patterns of migration and dispersal from the Mesolithic to early Middle Ages.
          <div class="bigspacer"></div>
          <h4>Example Papers</h4>

          <ol type = "1">
            <li><a href="/pdfs/papers/viking.pdf">Population genomics of the Viking world</a></li>
              <li><a href="/pdfs/papers/siberia.pdf">The population history of northeastern Siberia since the Pleistocene</a></li>
                <li><a href="/pdfs/papers/Americas.pdf">Early human dispersals within the Americas</a></li>
                  <li><a href="/pdfs/papers/137.pdf">137 ancient human genomes from across the Eurasian steppes</a></li>
                    <li><a href="/pdfs/papers/horse.pdf">The first horse herders and the impact of early Bronze Age steppe expansions into Asia</a></li>
                      <li><a href="/pdfs/papers/alaska.pdf">Terminal Pleistocene Alaskan genome reveals first founding population of Native Americans</a></li>
           </ol>
      <div class="bigspacer"></div>




          <h1>Viral Evolution</h1>
          <hr style="height:3px;border:none;color:#808088;background-color:#808088;" />
          <img src="/images/Picture5.png" style="float:right">

          We have on and off been working on viral evolution for the past several decades.
          After the emergence of SARS-CoV-2 we have focused renewed attention on this area to understand
           the evolution of the virus.  We expect viral evolution to continue to be an important area of
           focus in the lab.
           <div class="bigspacer"></div>
           <h4>Example Papers</h4>

           <ol type = "1">
            <li><a href="/pdfs/papers/synmut.pdf">Synonymous mutations and the molecular evolution of SARS-Cov-2 origins</a></li>
              <li><a href="/pdfs/papers/uncertainty.pdf">Assessing uncertainty in the rooting of the SARS-CoV-2 phylogeny</a></li>
                <li><a href="/pdfs/papers/point.pdf">A Bayesian framework for inferring the influence of sequence context on point mutations</a></li>
                  <li><a href="/pdfs/papers/passage.pdf">Passage adaptation correlates with the reduced efficacy of the influenza vaccine</a></li>
                    <li><a href="/pdfs/papers/rna.pdf">The evolutionary pathway to virulence of an RNA Virus</a></li>
            </ol>
            <div class="bigspacer"></div>
       <div class="bigspacer"></div>
       <div class="bigspacer"></div>









             <h1>Herpetogenomics</h1>
             <hr style="height:3px;border:none;color:#808088;background-color:#808088;" />
             <img src="/images/Picture6.jpg" style="float:right">

             We love reptiles and amphibians and maintain several research projects on their genomics
             and evolutionary genetics.  We are particularly fond of poison frogs, such as the lovely
             strawberry frog (<i>Oophaga pumillio</i>) shown in the picture.  In several projects, in collaboration
              with <a href="https://biology.ecu.edu/faculty/kyle-summers/">Kyle Summers</a> and
              <a href="http://www.rzlab.pitt.edu">Corinne Richards-Zawacki</a>, we are trying to elucidate the
               genetic basis of color variation in poison frogs. We also have ongoing research on the genomics
               of frogs living in high altitude.  Finally, we work on mapping of phenotypes and behavior in
               the side-blotched lizard.
               <div class="bigspacer"></div>
               <h4>Example Papers</h4>

               <ol type = "1">
                <li><a href="/pdfs/papers/leapfrog.pdf">Divergence, gene flow, and the origin of leapfrog geographic distributions: The history of colour pattern variation in Phyllobates poison‐dart frogs</a></li>
                  <li><a href="/pdfs/papers/takeover.pdf">Genomic takeover by transposable elements in the strawberry poison frog</a></li>
                    <li><a href="/pdfs/papers/plastic.pdf">The genetic basis of adaptation following plastic changes in coloration in a novel environment</a></li>
                </ol>
           <div class="bigspacer"></div>










             <h1>Phylogenetics and Comparative Analyses</h1>
             <hr style="height:3px;border:none;color:#808088;background-color:#808088;" />
             <img src="/images/Picture7.png" style="float:right">

             We have worked extensively on methods for phylogenetic analyses, particularly for using
              phylogentic trees for inferences in molecular evolution.  Many of the methods we have helped
              develop are implemented in
              the <a href="http://abacus.gene.ucl.ac.uk/software/#phylogenetic-analysis-by-maximum-likelihood-paml">PAML</a> package
                by <a href="http://abacus.gene.ucl.ac.uk">Ziheng Yang</a>.  We have also worked on modeling expression
                 level evolution along the lineages of a phylogeny and on various statistical methods for
                 imputation using phylogenies.  We are currently working on using large phylogenies
                 (>100k leaf nodes) for inference of evolutionary processes.
                 <div class="bigspacer"></div>
                 <h4>Example Papers</h4>

                 <ol type = "1">
                  <li><a href="/pdfs/papers/uncertainty.pdf">Assessing uncertainty in the rooting of the SARS-CoV-2 phylogeny</a></li>
                    <li><a href="/pdfs/papers/lizard.pdf">Phenotypic convergence is not mirrored at the protein level in a lizard adaptive radiation</a></li>
                      <li><a href="/pdfs/papers/anova.pdf">Phylogenetic ANOVA: The Expression Variance and Evolution Model for Quantitative Trait Evolution</a></li>
                  </ol>
             <div class="bigspacer"></div>
             <div class="bigspacer"></div>


             <h1>Methods for Environmental DNA Analyses</h1>
             <hr style="height:3px;border:none;color:#808088;background-color:#808088;" />
             <img src="/images/Picture8.jpg" style="float:right">

             We have on and off been working on DNA barcoding and analyses of environmental DNA over the past 15 years.
              This is an area of very active research in the lab at the moment.  We focus on developing better methods
               for species assignment and on developing methods for leveraging eDNA for population genetic analyses.
               <div class="bigspacer"></div>
               <h4>Example Papers</h4>

               <ol type = "1">
                <li><a href="/pdfs/papers/barcode.pdf">Fast phylogenetic DNA barcoding</a></li>
                  <li><a href="/pdfs/papers/assign.pdf">Statistical Assignment of DNA Sequences Using Bayesian Phylogenetics</a></li>
                    <li><a href="/pdfs/papers/greenland.pdf">Ancient Biomolecules from Deep Ice Cores Reveal a Forested Southern Greenland</a></li>
                      <li><a href="/pdfs/papers/sediment.pdf">Ancient DNA Chronology within Sediment Deposits: Are Paleobiological Reconstructions Possible and is DNA Leaching a Factor?</a></li>
                </ol>
           <div class="bigspacer"></div>
           <div class="bigspacer"></div>
           <div class="bigspacer"></div>
           <div class="bigspacer"></div>
           <div class="bigspacer"></div>






                <h1>Evolutionary Genomics of Domesticated Plants and Animals</h1>
                <hr style="height:3px;border:none;color:#808088;background-color:#808088;" />
                <img src="/images/Picture9.png" style="float:right">

                We have worked on multiple projects aimed at understanding evolutionary processes during domestication,
                particularly the way domestication selection has affected genetic variation and the consequences
                of admixture with wild species.  We are currently working particularly on rice and on betta fish.
                <div class="bigspacer"></div>
                <h4>Example Papers</h4>

                <ol type = "1">
                  <li><a href="/pdfs/papers/highalt.pdf">Ancient hybridization with an unknown population facilitated high-altitude adaptation of canids</a></li>
                    <li><a href="/pdfs/papers/bos.pdf">Pervasive introgression facilitated domestication and adaptation in the Bos species complex</a></li>
                      <li><a href="/pdfs/papers/wildrice.pdf">Asian wild rice is a hybrid swarm with extensive gene flow and feralization from domesticated rice</a></li>
                        <li><a href="/pdfs/papers/power.pdf">The power of inbreeding: NGS based GWAS of rice reveals convergent evolution during rice domestication</a></li>
                 </ol>
            <div class="bigspacer"></div>











                   <h1>Cancer Evolution and Cell Lineage Trees</h1>
                   <hr style="height:3px;border:none;color:#808088;background-color:#808088;" />
                   <img src="/images/Picture10.jpg" style="float:right">

                   Much of our previous work has focused on developing computational methods to elucidate
                    evolutionary changes in DNA or in phenotypes such as expression levels, along the branches
                     of evolutionary trees. We are now using similar computational techniques to elucidate
                     processes along the branches of cell lineage trees within an individual.  The process of
                      differentiation of cells during ontogeny/morphogenesis can be described by a tree, much
                      like an evolutionary tree.  Similarly, the process of mutation and cell differentiation
                       driving carcinogenesis can be described as an evolutionary process acting within an individual
                       in which the unit of evolution is a cell instead of an individual.  A new line of research in
                       the lab is the modeling and analysis of somatic mutations.
                       We have worked with <a href="http://www.bx.psu.edu/makova_lab/">Kateryna Markova</a> on
                       analyzing mitochondrial heteroplasmies and on developing probabilistic models of the spread of
                        somatic mutations during ontogenesis.  These models allow us to take advantage of large
                        NGS data sets to test hypotheses regarding differential proliferation of mutations in different
                         tissues and on natural selection acting on the population of cells within an individual.
                          We are also working on modeling single-cell sequencing data from tumors to provide better
                          computational methods for analyzing tumor evolution.
                          <div class="bigspacer"></div>
                          <h4>Example Papers</h4>

                          <ol type = "1">
                           <li><a href="/pdfs/papers/hair.pdf">Pronounced somatic bottleneck in mitochondrial DNA of human hair</a></li>
                            <li><a href="/pdfs/papers/bottle.pdf">Bottleneck and selection in the germline and maternal age influence transmission of mitochondrial DNA in human pedigrees</a></li>
                              <li><a href="/pdfs/papers/hetero.pdf">A population phylogenetic view of mitochondrial heteroplasmy</a></li>
                           </ol>
                      <div class="bigspacer"></div>
